packages/fluent/scss/popover/_variables.scss

```diff
@@ -0,0 +1,89 @@
+@use "../core/" as *;
+@use "../card/_variables.scss"as *;
+@use "../window/_variables.scss"as *;
+@use "../dialog/_variables.scss"as *;
+@use "../popup/_variables.scss"as *;
+
+/// Border width around the popover.
+/// @group popover
+$kendo-popover-border-width: $kendo-dialog-border-width !default;
+/// Border style around the popover.
+/// @group popover
+$kendo-popover-border-style: solid !default;
+/// Border radius around the popover.
+/// @group popover
+$kendo-popover-border-radius: $kendo-window-border-radius !default;
+/// Font size of the popover.
+/// @group popover
+$kendo-popover-font-size: $kendo-window-font-size !default;
+/// Font family of the popover.
+/// @group popover
+$kendo-popover-font-family: $kendo-window-font-family !default;
+/// Line height of the popover.
+/// @group popover
+$kendo-popover-line-height: $kendo-window-line-height !default;
+
+/// Background color of the popover.
+/// @group popover
+$kendo-popover-bg: var( --kendo-component-bg, initial ) !default;
+/// Text color of the popover.
+/// @group popover
+$kendo-popover-text:  var( --kendo-component-text, initial ) !default;
+/// Border color of the popover.
+/// @group popover
+$kendo-popover-border:  var( --kendo-component-border, initial ) !default;
+/// Shadow of the popover.
+/// @group popover
+$kendo-popover-shadow: $kendo-popup-shadow !default;
+
+/// Horizontal padding of the popover header.
+/// @group popover
+$kendo-popover-header-padding-x: $kendo-dialog-titlebar-padding-x !default;
+/// Vertical padding of the popover header.
+/// @group popover
+$kendo-popover-header-padding-y: $kendo-dialog-titlebar-padding-y !default;
+/// Border width of the popover header.
+/// @group popover
+$kendo-popover-header-border-width: $kendo-dialog-titlebar-border-width !default;
+/// Border style of the popover header.
+/// @group popover
+$kendo-popover-header-border-style: $kendo-popover-border-style !default;
+/// Background color of the popover header.
+/// @group popover
+$kendo-popover-header-bg: $kendo-dialog-titlebar-bg !default;
+/// Text color of the popover header.
+/// @group popover
+$kendo-popover-header-text: $kendo-dialog-titlebar-text !default;
+/// Border color of the popover header.
+/// @group popover
+$kendo-popover-header-border: $kendo-dialog-titlebar-border !default;
+
+/// Horizontal padding of the popover body.
+/// @group popover
+$kendo-popover-body-padding-x: $kendo-window-inner-padding-x !default;
+/// Vertical padding of the popover body.
+/// @group popover
+$kendo-popover-body-padding-y: $kendo-card-body-padding-y !default;
```

Comment:
these seem a bit inconsistent to me- the vertical padding is connected to the window-inner-padding-x, while the vertical to the card-padding-y. I would double check with the designer and rather make them the same as the paddings of the card (as in the rest of the themes)

Reply:
We had discussed the case with the designer and the conclusion was the following: as Popover in MS Fluent theme looks like the Dialog, it gets paddings from such widget. In our case Dialog applies some paddings from .k-window, so it has no variables for all of them. In the Popover we do not have .k-window className, so we should import some padding variables from the Window widget. For that reason we use paddings from the both widgets.
I have fixed/replaced padding imported from the Card.

Reply:
This sounds rather strange to me. The card has the same composite elements as the Card (header, content, actions) and also identical colors and padding for the same. If we can set all variables in the Popover to inherit from Card variables, I think this would be a better solution implementation wise and would reduce the complexity and dependencies of the component.

Reply:
MS Fluent Popover uses the same paddings as Dialog and Window, not as Card.